include/my_rdtsc.h

```diff
@@ -42,6 +42,8 @@
 */
 struct my_timer_unit_info
 {
+  /** Function */
```

Comment:
Function for what?

Reply:
As a suggestion, why not name the member based on what the function returns, such as `time()` or `get_time()` or `get_millis()`

Reply:
After more reading, the unit seems to differ, so it's actually cycles, right?

Reply:
Usually cycles but could be a higher milliseconds, microseconds if cycles in NA.

Its a measurements, of what maps to the frequency. So `get_measure()`?